scania-digital-design-system/sdds
# chore: fix importing md files in stories not working

Importing md files in stories previously gave a typescript error. Now it shouldn't.

**How to test**  

Open the project in your editor, make sure eslint and prettier is installed and configured.
Verify there are no errors in story files that import markdown files.

Verify that npm packages are still excluding the stories.

**Screenshots**  
No more error:
![Screenshot 2023-01-03 at 12 01 42](https://user-images.githubusercontent.com/8556022/210344982-161aedcc-b76b-41a0-8e4b-2bbe3a4e2a6d.png)


**Additional context**  
PR that introduced issue: #727
